src/config.ts

```diff
@@ -161,6 +161,7 @@ function getConfig() {
       useDeprecatedAliasFlag: configOptions.get<boolean>('jackIn.useDeprecatedAliasFlag'),
     },
     enableClojureLspOnStart: configOptions.get<boolean>('enableClojureLspOnStart'),
+    projectRootsSearchExclude: configOptions.get<string[]>('projectRootsSearchExclude'),
```

Comment:
Oh! I missed one thing! We should probably add a default here so that it satisfies the gods of `strictNullChecks`:

```
configOptions.get<string[]>('projectRootsSearchExclude') ?? []
```

Reply:
Thanks. I used the default value signature of the `get`:

```ts
configOptions.get<string[]>('projectRootsSearchExclude', []) 
```

Does that also satisfy the rule? Less syntax to parse...

Reply:
ooooh, I didn't know about that one! I think that should fix it? and If not I can fix it later when i'm fixing null check things 😊